src/components/AccountDetails/index.tsx

```diff
@@ -230,6 +231,9 @@ export default function AccountDetails({
   const theme = useContext(ThemeContext)
   const dispatch = useAppDispatch()
 
+  const isMetaMask = !!window.ethereum?.isMetaMask
+  const isCoinbaseWallet = !!window.ethereum?.isCoinbaseWallet
```

Comment:
Make another variable to use in the tsx, so that the intent is clearer in code:

const isMobileWallet

Reply:
good point